neo/SmartContract/Manifest/ContractManifest.cs

```diff
@@ -91,20 +90,7 @@ public static ContractManifest CreateDefault(UInt160 hash)
         /// <returns>Return true or false</returns>
         public bool CanCall(ContractManifest manifest, string method)
         {
-            if (Groups.Any(a => manifest.Groups.Any(b => a.PubKey.Equals(b.PubKey))))
-            {
-                // Same group
-                return true;
-            }
-
-            if (manifest.Trusts != null && !manifest.Trusts.IsWildcard && !manifest.Trusts.Contains(Hash))
```

Comment:
now `Trusts` is not used

Reply:
`trusts` is for displaying warnings by client or wallet.

Reply:
Then `trusts` and `safeMethods` is only for user interfaces?

Reply:
Yes.